src/Jenssegers/Mongodb/Relations/EmbedsMany.php

```diff
@@ -265,25 +265,36 @@ protected function associateExisting($model)
     }
 
     /**
-     * Get a paginator for the "select" statement.
-     * @param int $perPage
-     * @return \Illuminate\Pagination\AbstractPaginator
+     * @param null $perPage
+     * @param array $columns
+     * @param string $pageName
+     * @param null $page
+     * @return \Illuminate\Contracts\Pagination\LengthAwarePaginator
      */
-    public function paginate($perPage = null)
+    public function paginate($perPage = null, $columns = ['*'], $pageName = 'page', $page = null)
     {
         $page = Paginator::resolveCurrentPage();
```

Comment:
```suggestion
        $page = $page ?: Paginator::resolveCurrentPage($pageName);
```

https://github.com/laravel/framework/blob/6.x/src/Illuminate/Database/Eloquent/Builder.php#L716